site/data/tables.sql

```diff
@@ -205,6 +205,8 @@ CREATE TABLE gradeable_component (
 CREATE TABLE gradeable_component_data (
     gc_id integer NOT NULL,
     gd_id integer NOT NULL,
+    gcd_grader_id character varying(255) NOT NULL,
+    gcd_grade_time timestamp(6) without time zone NOT NULL,
```

Comment:
either we have to allow these to be not null or you'll need to update the inserts for test grading and homework grading. 

I think I prefer the former for now (remove the NOT NULL constraint) and to create a new issue that says to "Make gradeable_component_data.gcd_grader_id NOT NULL and foreign key" with a link to the other issue I created to track implementation of this feature.